% \documentclass[notes]{beamer}
% \documentclass[notes=hide]{beamer}
% \documentclass[notes=only]{beamer}
\documentclass[handout]{beamer}

\usepackage{pgfpages}
\pgfpagesuselayout{2 on 1}[a4paper,border shrink=3mm]

\usepackage[utf8]{inputenc}
\usepackage[T1]{fontenc}
\usepackage{graphicx}
\usepackage{natbib}
\usepackage{xcolor,colortbl}

\definecolor{LightCyan}{rgb}{0.88,1,1}

\usetheme{riku}

\def\logo{
	{\includegraphics[width=20mm]{logo}}
}

\title[BMI and Cancer Pathway]{Body Mass Index and Pathway Dysregulation in Cancer}
\subtitle{Committee Meeting 2016}
\author[Riku Takei]{Riku Takei (MSc)\\Supervisor: Assoc. Prof. Mik Black}
\institute{Department of Biochemistry, University of Otago}

\begin{document}

{
	\setbeamertemplate{footline}{}
	\begin{frame}[noframenumbering]
		\titlepage
	\end{frame}
}

\begin{frame}
\frametitle{Introduction}
\begin{center}
	\includegraphics[scale=0.35]{./images/obesityworldmap}
\end{center}
\end{frame}

\begin{frame}
\frametitle{Introduction}
\begin{center}
	\includegraphics[width=0.48\paperwidth]{./images/ob1999}\\
	1993\\
	\includegraphics[width=0.48\paperwidth]{./images/ob2014}\\
	2014
\end{center}
\end{frame}

\begin{frame}
	\frametitle{Body Mass Index and Cancer}
	\begin{columns}
		\column{0.6\textwidth}
		{\footnotesize
			Cancers are caused by dysregulation in various pathways/mechanisms that allow them to survive and proliferate
			\begin{itemize}
				\item Commonly know as the Hallmarks of Cancer \citep{Hanahan2011}
			\end{itemize}
			Many reports on obesity being a major risk for cancer and poor prognosis
		}
		\column{0.5\textwidth}
		\includegraphics[width=0.45\paperwidth]{./images/cancerhallmark}
	\end{columns}
\end{frame}

\begin{frame}
	\frametitle{Obesity-associated Genetic Signatures}
	Creighton \textit{et al.} (2012)
	\begin{itemize}
		\item Identified obesity-associated genetic signatures in breast cancer patients
		\item Showed that some compounds were able to specifically regulate the expression of these genetic signatures (potential for targeted therapy)
	\end{itemize}
	Fuentes-Mattei \textit{et al.} (2014)
	\begin{itemize}
		\item Identified obesity-associated genetic signatures in ER+ breast cancer patients
		\item Direct evidence of how obesity/adiposity contributes to tumour growth (using mouse model)
	\end{itemize}
\end{frame}

% \begin{frame}
% 	\frametitle{Genetic Signatures Across Multiple Cancer Types}
% 	\begin{center}
% 		\includegraphics[width=0.85\paperwidth]{./images/multicancersignature}
% 	\end{center}
% \end{frame}

\begin{frame}
	\frametitle{Aims}
		\begin{enumerate}
			\item Investigate the association between patient BMI and tumour molecular characteristics
			\item Determine whether pathway commonalities exist across tumours that arise from other tissues
		\end{enumerate}
\end{frame}

\begin{frame}{Progress Overview}
	Where I was at the last committee meeting:
	{\footnotesize
		\begin{itemize}
			\item Applied Creighton metagene to ICGC data
			\item Applied Fuentes-Mattei metagene to Creighton \textit{et al.} and ICGC data
			\item Found no association with BMI status
		\end{itemize}
	}
	\vskip2ex
	What I have done since the last committee meeting:
	{\footnotesize
		\begin{itemize}
			\item Gene expression analysis using Creighton \textit{et al.} data
			\item Gene expression analysis using ICGC data
			\item Pathway enrichment analysis using ICGC data
			\item Comparison of the obesity metagenes with other pathway metagenes from Gatza \textit{et al.} (preliminary results)
		\end{itemize}
	}
\end{frame}

\begin{frame}
	\frametitle{Methods}
	Download sequence (RNA-seq) data and clinical data from publicly available databases
	{\footnotesize
		\begin{itemize}
			\item The Cancer Genome Atlas (TCGA)
			\item International Cancer Genome Consortium (ICGC)
		\end{itemize}
	}
	\vskip2ex
	\includegraphics[width=0.4\textwidth]{./images/tcga}
	\hfill
	\includegraphics[width=0.4\textwidth]{./images/icgc}
\end{frame}

\begin{frame}{Clinical Data}
	Download all the clinical data available from TCGA databases
	\begin{itemize}
		\item {\footnotesize Clinical data available for 33 cancer types}\\
	\end{itemize}
	\vskip2ex
	Only interested in cancer types that have both height and weight data\\
	\begin{equation*}
		BMI = \frac{Weight (kg)}{Height^2 (m^2)}
	\end{equation*}
\end{frame}

\begin{frame}{Clinical Data with Patient BMI}
	Out of 33 cancer types, 14 types had both height and weight data\\
	{\footnotesize
		\begin{itemize}
			\item 8 cancer types had RNA-seq data\\
		\end{itemize}
	}
	\vskip2ex
	\begin{center}
		\begin{tabular}{lcccl}
			& Lean & Overweight & Obese & total \\
			\rowcolor{LightCyan}
			BLCA & 103  & 97         & 61    & 261\\
			CESC & 85   & 62         & 77    & 224   \\
			\rowcolor{LightCyan}
			COAD & 75   & 80         & 71    & 226   \\
			KIRP & 26   & 53         & 45    & 124\\
			\rowcolor{LightCyan}
			LIHC & 140  & 75         & 49    & 264\\
			READ & 22   & 36         & 15    & 73\\
			\rowcolor{LightCyan}
			SKCM & 72   & 73         & 73    & 218\\
			UCEC & 90   & 106        & 286   & 482\\
			\hline
			\rule{0pt}{2.25ex}Total & 613 & 582 & 677 & 1872\\
		\end{tabular}
	\end{center}
\end{frame}

\begin{frame}{Validation of Methodology -- Creighton \textit{et al.} (2012)}
	103 samples used in the study, 799 obesity-associated gene probes (695 unique genes) identified\\
	{\footnotesize
		\begin{itemize}
			\item These gene probes were used to assign an obesity-associated gene signature score to each sample
		\end{itemize}
	}
	\begin{center}
		\includegraphics[width=0.6\paperwidth]{./images/creighton}
	\end{center}
\end{frame}

\begin{frame}{Validation of Methodology -- Creighton \textit{et al.} (2012)}
	Normalised the data using RMA normalisation method (\texttt{affy} package)\\
	\vskip2ex
	Metagene was obtained from SVD using the genes that were common in both Creighton \textit{et al.} and other cancer data sets\\
	\begin{center}
		\includegraphics[width=0.6\paperwidth]{./images/creighton}
	\end{center}
\end{frame}

\begin{frame}{Singular Value Decomposition (SVD) recap}
	For each sample in the data set, you need to assign a ``score'' that represents the overall gene expression of the gene set of interest
	\begin{itemize}
		\item i.e. a score that represent the 799 obesity-associated gene probes identified by Creighton \textit{et al.}
	\end{itemize}
	\vskip2ex
	You can do this by applying SVD to the data set of interest and get a metagene out from this
\end{frame}

\begin{frame}{\normalsize Creighton \textit{et al.} metagene correlates with Gene Expression}
	\begin{center}
		\includegraphics[width=0.45\paperwidth]{./images/crge1}
	\end{center}
\end{frame}

\begin{frame}{Creighton \textit{et al.} metagene correlates with BMI}
	\hspace{-4.0ex}
	\includegraphics[width=0.45\paperwidth]{./images/crmetaclin0}
	\includegraphics[width=0.45\paperwidth]{./images/crmetaclin1}
\end{frame}

\begin{frame}{Transformation Matrix}
	Singular Value Decomposition of a dataset X gives:
	\begin{equation*}
		X = UDV'
	\end{equation*}
	Rearranging this equation gives:
	\begin{equation*}
		V' = D^{-1}U'X
	\end{equation*}
	Where $D^{-1}U'$ is the transformation matrix and $V'$ is the metagene for the data set X
	{\footnotesize
		\begin{itemize}
			\item This means that the transformation matrix can be applied to other data sets to obtain the metagene for those data sets
		\end{itemize}
	}
\end{frame}

\begin{frame}{Metagene in ICGC data}
	\includegraphics[width=0.23\paperwidth]{./images/crtcga2}
	\includegraphics[width=0.23\paperwidth]{./images/crtcga7}
	\includegraphics[width=0.23\paperwidth]{./images/crtcga12}
	\includegraphics[width=0.23\paperwidth]{./images/crtcga17}\\
	\includegraphics[width=0.23\paperwidth]{./images/crtcga22}
	\includegraphics[width=0.23\paperwidth]{./images/crtcga27}
	\includegraphics[width=0.23\paperwidth]{./images/crtcga32}
	\includegraphics[width=0.23\paperwidth]{./images/crtcga37}
\end{frame}

\begin{frame}{Does the metagene correlate with BMI?}
	\includegraphics[width=0.23\paperwidth]{./images/crtcga3}
	\includegraphics[width=0.23\paperwidth]{./images/crtcga8}
	\includegraphics[width=0.23\paperwidth]{./images/crtcga13}
	\includegraphics[width=0.23\paperwidth]{./images/crtcga18}\\
	\includegraphics[width=0.23\paperwidth]{./images/crtcga23}
	\includegraphics[width=0.23\paperwidth]{./images/crtcga28}
	\includegraphics[width=0.23\paperwidth]{./images/crtcga33}
	\includegraphics[width=0.23\paperwidth]{./images/crtcga38}
\end{frame}

\begin{frame}{Obesity-associated genes from Fuentes-Mattei \textit{et al.}}
	\begin{columns}
		\column{0.5\textwidth}
		137 ER+ breast cancer samples in the study, 130 unique genes identified\\
		\vskip2ex
		Data were normalised  with RMA normalisation method\\
		\vskip2ex
		Metagene was formed from the common genes between the Fuentes-Mattei \textit{et al.} and other cancer data sets (116 genes)
		\column{0.5\textwidth}
		\includegraphics[width=0.45\paperwidth]{./images/fmcircos}
	\end{columns}
\end{frame}

% \begin{frame}{Can the FM metagene predict BMI status?}
% 	If both Creighton \textit{et al.} and Fuentes-Mattei \textit{et al.} found obesity-associated genetic signature, then these signatures should be similar
% 	{\footnotesize
% 		\begin{itemize}
% 			\item Therefore should be able to separate the samples based on BMI
% 		\end{itemize}
% 	}
% 	\vskip2ex
% 	No BMI information is available for Fuentes-Mattei \textit{et al.} data set, so you need to transform the Creighton \textit{et al.} with the metagene\\
% 	{\footnotesize
% 		\begin{itemize}
% 			\item Check whether the metagene correlates with BMI status
% 		\end{itemize}
% 	}
% \end{frame}

\begin{frame}{FM metagene in Creighton \textit{et al.} data}
	\begin{center}
		\includegraphics[width=0.45\paperwidth]{./images/fmmetacr3}
	\end{center}
\end{frame}

\begin{frame}{FM metagene in Creighton \textit{et al.} data}
	\includegraphics[width=0.45\paperwidth]{./images/fmmetacr4}
	\includegraphics[width=0.45\paperwidth]{./images/fmmetacr5}
\end{frame}

\begin{frame}{FM metagene in other cancer types}
	\includegraphics[width=0.23\paperwidth]{./images/fmmetaheat2}
	\includegraphics[width=0.23\paperwidth]{./images/fmmetaheat7}
	\includegraphics[width=0.23\paperwidth]{./images/fmmetaheat12}
	\includegraphics[width=0.23\paperwidth]{./images/fmmetaheat17}\\
	\includegraphics[width=0.23\paperwidth]{./images/fmmetaheat22}
	\includegraphics[width=0.23\paperwidth]{./images/fmmetaheat27}
	\includegraphics[width=0.23\paperwidth]{./images/fmmetaheat32}
	\includegraphics[width=0.23\paperwidth]{./images/fmmetaheat37}
\end{frame}

\begin{frame}{FM metagene and BMI}
	\includegraphics[width=0.23\paperwidth]{./images/fmmetabmi3}
	\includegraphics[width=0.23\paperwidth]{./images/fmmetabmi8}
	\includegraphics[width=0.23\paperwidth]{./images/fmmetabmi13}
	\includegraphics[width=0.23\paperwidth]{./images/fmmetabmi18}\\
	\includegraphics[width=0.23\paperwidth]{./images/fmmetabmi23}
	\includegraphics[width=0.23\paperwidth]{./images/fmmetabmi28}
	\includegraphics[width=0.23\paperwidth]{./images/fmmetabmi33}
	\includegraphics[width=0.23\paperwidth]{./images/fmmetabmi38}
\end{frame}

\begin{frame}{Summary 1}
	Metagene from Creighton \textit{et al.} correlated with gene expression and patient BMI in Creighton data set
	\vskip2ex
	Metagene from Creighton \textit{et al.} correlated with gene expression but not the patient BMI in ICGC data
	\vskip2ex
	Metagene from Fuentes-Mattei \textit{et al.} correlated with gene expression but not the patient BMI in both Creighton \textit{et al.} and ICGC data
\end{frame}

\begin{frame}{Gene Expression Analysis}
	Maybe the obesity-associated genes identified by either group were not related to obesity, but related to a different clinical variable
	\vskip2ex
	Does the differentially expressed genes between the obese and non-obese groups (after correcting for other clinical variables) similar to the obesity-associated genes?
	{\footnotesize
		\begin{itemize}
				\item How do these DEGs compare with the Creighton's or Fuentes-Mattei's obesity-associated genes?
		\end{itemize}
	}
\end{frame}

\begin{frame}{Creighton \textit{et al.} data revisited}
	Gene expression analysis was carried out on the Creighton \textit{et al.} data
	{\footnotesize
		\begin{itemize}
			\item in their paper, they found 799 gene probes with p \textless{} 0.01 and fold change \textgreater{} 1.2
		\end{itemize}
	}
	\vskip2ex
	1781 differentially expressed genes (p \textless{} 0.01) were found between obese vs. non-obese
	{\footnotesize
		\begin{itemize}
			\item very few genes had fold change \textgreater{} 1.2 (132 genes)
			\item no genes were found when p-value was adjusted
			\item top 799 genes were chosen from this list
		\end{itemize}
	}
	\vskip2ex
	239 gene probes were present in the top 799 gene probes that I have found
\end{frame}

\begin{frame}{Residual data}
	All of the non-BMI clinical variables were controlled for in the linear model
	{\footnotesize
		\begin{itemize}
			\item Age, ethnicity, menopause status, grade, LN/ER/PR/HER2 status
		\end{itemize}
	}
	\vskip2ex
	Residual data (or remaining data after controlling for the variables) was used to find the DEGs
	\vskip2ex
	Again, more than 799 gene probes showed up (1104 genes)
	{\footnotesize
		\begin{itemize}
			\item 168 genes from Creighton \textit{et al.} were found in the top 799 genes
		\end{itemize}
	}
\end{frame}

\begin{frame}{DEGs in residual data}
	\begin{center}
		\includegraphics[width=0.50\paperwidth]{./images/venn0}
	\end{center}
\end{frame}

\begin{frame}{Completely remove the effect of ethnicity}
	African American patients are more likely to be obese than Cauacasian patients
	\vskip2ex
	Removed the effect of ethnicity by removing the African American patients' data (total of 77 patient data to work with)
	{\footnotesize
		\begin{itemize}
			\item All of the other clinical variables were adjusted as before
		\end{itemize}
	}
	\vskip2ex
	Top 799 genes were identified in this data set (both raw and residual)
	{\footnotesize
		\begin{itemize}
			\item  148 and 92 genes were in the top 799 genes in the raw and residual data, respectively
		\end{itemize}
	}
\end{frame}

\begin{frame}{DEGs in Cauacasian-only data}
	\begin{center}
		\includegraphics[width=0.50\paperwidth]{./images/venn1}
	\end{center}
\end{frame}

\begin{frame}{Metagene Analysis}
	All of the new/novel gene probe sets (8 in total) were used to create metagenes and used to transform other cancer types
	\begin{itemize}
		\item Checked in Creighton \textit{et al.} data
		\item Transferred into other cancer data sets
	\end{itemize}
\end{frame}

\begin{frame}{Initial results -- Checking in Creighton \textit{et al.} data}
	\includegraphics[width=0.23\paperwidth]{./images/degmetacr2}
	\includegraphics[width=0.23\paperwidth]{./images/degmetacr7}
	\includegraphics[width=0.23\paperwidth]{./images/degmetacr12}
	\includegraphics[width=0.23\paperwidth]{./images/degmetacr17}\\
	\includegraphics[width=0.23\paperwidth]{./images/degmetacr22}
	\includegraphics[width=0.23\paperwidth]{./images/degmetacr27}
	\includegraphics[width=0.23\paperwidth]{./images/degmetacr32}
	\includegraphics[width=0.23\paperwidth]{./images/degmetacr37}
\end{frame}

\begin{frame}{Initial results -- Checking in Creighton \textit{et al.} data}
	\includegraphics[width=0.23\paperwidth]{./images/degmetacr3}
	\includegraphics[width=0.23\paperwidth]{./images/degmetacr8}
	\includegraphics[width=0.23\paperwidth]{./images/degmetacr13}
	\includegraphics[width=0.23\paperwidth]{./images/degmetacr18}\\
	\includegraphics[width=0.23\paperwidth]{./images/degmetacr23}
	\includegraphics[width=0.23\paperwidth]{./images/degmetacr28}
	\includegraphics[width=0.23\paperwidth]{./images/degmetacr33}
	\includegraphics[width=0.23\paperwidth]{./images/degmetacr38}
\end{frame}

\begin{frame}{Do the metagenes correlate with gene expression in ICGC data?}
	\includegraphics[width=0.23\paperwidth]{./images/rawobsmeta2}
	\includegraphics[width=0.23\paperwidth]{./images/resobsmeta2}
	\includegraphics[width=0.23\paperwidth]{./images/caobsmeta2}
	\includegraphics[width=0.23\paperwidth]{./images/caresobsmeta2}\\
	\includegraphics[width=0.23\paperwidth]{./images/rawcrolmeta2}
	\includegraphics[width=0.23\paperwidth]{./images/rescrolmeta2}
	\includegraphics[width=0.23\paperwidth]{./images/cacrolmeta2}
	\includegraphics[width=0.23\paperwidth]{./images/carescrolmeta2}
\end{frame}

\begin{frame}{Do the metagenes correlate with BMI?}
	\includegraphics[width=0.23\paperwidth]{./images/rawobsclin3}
	\includegraphics[width=0.23\paperwidth]{./images/resobsclin3}
	\includegraphics[width=0.23\paperwidth]{./images/caobsclin3}
	\includegraphics[width=0.23\paperwidth]{./images/caresobsclin3}\\
	\includegraphics[width=0.23\paperwidth]{./images/rawcrolclin3}
	\includegraphics[width=0.23\paperwidth]{./images/rescrolclin3}
	\includegraphics[width=0.23\paperwidth]{./images/cacrolclin3}
	\includegraphics[width=0.23\paperwidth]{./images/carescrolclin3}
\end{frame}

\begin{frame}{Do the metagenes correlate with each other?}
	\begin{center}
		\includegraphics[width=0.45\paperwidth]{./images/cor0}
		\includegraphics[width=0.45\paperwidth]{./images/cor1}
	\end{center}
\end{frame}

\begin{frame}{Metagenes are correlating with GE but not BMI}
	All of the metagenes found so far have correlated with the overall gene expression of the samples, but not with the sample BMI
	\vskip2ex
	The metagenes are picking up gene signatures  that are common across all cancer types, and not related to BMI
	\vskip2ex
	\textbf{So what signature/mechanism is the metagene picking up?}
\end{frame}

\begin{frame}{Pathway signatures from Gatza \textit{et al.} (2010)}
	Made metagenes from the pathway signatures from the Gatza \textit{et al.} paper in ICGC data
	\vskip2ex
	Compared these pathway metagenes with the BMI metagenes
	{\footnotesize
		\begin{itemize}
			\item Is the BMI metagene similar to any of the pathway signatures?
		\end{itemize}
	}
	\vskip2ex
	All of the samples from the ICGC data were used (1872 samples in total)
\end{frame}

\begin{frame}{Results}
	\begin{center}
		\includegraphics[width=0.90\paperwidth, height=0.60\paperheight]{./images/allmeta1}
	\end{center}
\end{frame}

\begin{frame}{Results}
	\begin{center}
		\includegraphics[width=0.50\paperwidth]{./images/allmeta2}
	\end{center}
\end{frame}

\begin{frame}{Summary 2}
	Metagenes found in Creighton \textit{et al.} data correlates with GE and BMI status in Creighton data, but not in  the ICGC data
	\vskip2ex
	The metagenes do not correlate with any other pathway metagenes
	\vskip2ex
	\textbf{Are there any common genes and/or pathways being dysregulated across all of the cancer types?}
\end{frame}

\begin{frame}{Common genes across multiple cancer types}
	DEGs between the obese and non-obese groups were identified for each cancer types
	\vskip2ex
	For each cancer types, samples were randomly split into two groups (same proportion as obese vs. non-obese samples)
	\vskip2ex
	{\footnotesize
		\begin{tabular}{ccccccccc}
			& \multicolumn{8}{c}{No. of cancer types the gene overlapped}\\
            \rule{0pt}{2.25ex}                    & 1    & 2    & 3   & 4  & 5 & 6 & 7 & 8 \\
			\hline
			\rule{0pt}{2.25ex}Obese vs. Non-obese & 6865 & 1907 & 289  & 26   & 1 & 0 & 0 & 0 \\
			Randomly split groups                 & 6202 & 1488 & 198  & 18   & 1 & 0 & 0 & 0 \\
			No. of genes found                    & 5012 & 870  & 89   & 6    & 0 & 0 & 0 & 0 \\
			on average                            &      &      &      &      &   &   &   & \\
			Empirical FDR                         & 0.73 & 0.46 & 0.31 & 0.23 & 0 & 0 & 0 & 0 \\
		\end{tabular}
	}
\end{frame}

\begin{frame}{Pathway Enrichment Analysis}
	Maybe each cancer type have a different set of genes activated
	\vskip2ex
	Are there any pathways enriched by these genes?
	\vskip2ex
	If so, is it common across other cancer types?
	\vskip2ex
	How relevant is the pathway to obesity and/or cancer?
	\vskip2ex
	Can we use it to narrow down our search for obesity-associated signature?
\end{frame}

\begin{frame}{Pathway Enrichment Analysis}
	Gene Ontology (GO) database was used to check for the function(s) being enriched by the DEGs in each cancer type (between obese and non-obese groups)
	\vskip2ex
	Used a rank-based approach and controlled the error rate using FDR
	\vskip2ex
	\begin{center}
		\textbf {No pathways showed up as significantly enriched}
	\end{center}
\end{frame}

\begin{frame}{Summary}
	\textbf{Creighton/FM metagene analysis}
	\begin{itemize}
		\item Both Creighton and FM metagenes correlated with the overall gene expression in any cancer data set
			\item The metagene score does not correlate with the BMI status of the samples
	\end{itemize}
	\textbf{Gene expression and metagene analysis}
	\begin{itemize}
		\item Again, all of the gene sets found correlated with the overall gene expression in any data set, but none of them correlated with the BMI status
	\end{itemize}
	\textbf{Pathway enrichment analysis}
	\begin{itemize}
		\item There were no pathway significantly enriched in any cancer type
	\end{itemize}
\end{frame}

\begin{frame}{Discussion}
	Different methods used to find DEGs
	{\footnotesize
		\begin{itemize}
			\item Found more genes than Creighton \textit{et al.} because they were using different programs and normalisation method in their analyses
				\begin{itemize}
					\item Logged the data instead of RMA normalisation
				\end{itemize}
		\end{itemize}
	}
	\vskip2ex
	Tumour heterogeneity within and between samples
	{\footnotesize
		\begin{itemize}
			\item Not finding any significant genes or pathways because the samples are so different from one another
		\end{itemize}
	}
	\vskip2ex
	Tumour sample quality and size
	{\footnotesize
		\begin{itemize}
			\item Picking up too much noise and/or don't have enough samples to find anything significant
		\end{itemize}
	}
\end{frame}

\begin{frame}{Discussion -- Tumour heterogeneity}
	Each sample has an obesity driven pathway dysregulated, as well as different ``core'' pathways that lead to cancer progression
	{\footnotesize
		\begin{itemize}
			\item Each data set will have more/less of these samples
			\item Therefore picking up DEGs that are significant in that one particular data set but not the other data sets
		\end{itemize}
	}
	\vskip2ex
	Each data set will have a different collection of samples, which in turn have a different collection of dysregulated pathways
	{\footnotesize
		\begin{itemize}
			\item Hence DEGs show up, but not the pathways
		\end{itemize}
	}
\end{frame}

\begin{frame}{Conclusion}
	Obesity-associated genetic signatures cannot be used in other cancer types, nor in the same cancer type with different sample population
	\vskip2ex
	Gene expression and pathway enrichment analyses showed that there may not be any obesity-associated signature across multiple cancer types
\end{frame}

\begin{frame}{Future Directions}
	Explore the biology of the ``obesity-associated'' metagene
	{\footnotesize
		\begin{itemize}
			\item Use other pre-defined pathway signatures and see if any of them correlates with the metagene
		\end{itemize}
	}
	\vskip2ex
	Effect of BMI on methylation status in different cancer types
\end{frame}

\begin{frame}{Acknowledgement}
	\begin{columns}
		\column{0.5\textwidth}
		My supervisor: Mik\\
		Members of the Black lab\\
		Cancer Genetics Lab\\
		SYSKA/Mozilla Study Group\\
		ICGC and TCGA\\
		\column{0.5\textwidth}
		\includegraphics[width=0.3\paperwidth]{./images/tcga}\\
		\includegraphics[width=0.3\paperwidth]{./images/icgc}
	\end{columns}
\end{frame}

\begin{frame}
	\frametitle{References}
	\bibliographystyle{BiocRefStyle}
	\bibliography{/home/riku/Documents/References/BibTeX/MSc}
\end{frame}

\begin{frame}{test}
	\citet{Fuentes-Mattei2014}\\
	\citet{Creighton2012}\\
	\citet{Alexandrov2013}
	\citet{Gatza2010}
\end{frame}

\end{document}
