\documentclass[12pt, a4paper]{article}

\usepackage{graphicx}
\usepackage{subcaption}
\usepackage[labelfont=bf,font=footnotesize]{caption}
\usepackage[utf8]{inputenc}
\usepackage[T1]{fontenc}
\usepackage{natbib}
\usepackage{geometry}
\usepackage{fixltx2e}
\usepackage{enumitem}
\usepackage{amsmath}
\usepackage{mathptmx}

\geometry{
    a4paper,
    left=2.5cm,
    right=2.5cm,
    top=2.5cm,
    bottom=2.5cm
}

\linespread{1.3}

\defcitealias{TCGA}{NCI and NHGRI, 2015}
\defcitealias{ICGC}{ICGC, 2015}
\defcitealias{GEO}{NCBI, 2015}

\setlength{\bibsep}{5pt plus 0.3ex}

\graphicspath{{./CommitteeMeeting2015/}}

\begin{document}

\begin{center}
\Large{\textbf{Body Mass Index and Pathway Dysregulation in Cancer}}\\
%\end{center}

%\begin{center}
\normalsize{
    Riku Takei (MSc)\\
        Supervisor: Assoc. Prof. Mik Black
}
\end{center}

\vspace{-30pt}

\subsection*{50 Word Summary}

Recent breast cancer studies have demonstrated the existence of that there were obesity-specific genetic signatures.
In this work, these signatures were investigated in other cancer types.
Obesity genes identified by previous studies did not correlate with obesity in any other cancer types investigated.


\vspace{-11pt}

\subsection*{Introduction}

It is now commonly understood that cancers are caused by dysregulation of various mechanisms or pathways (known as the ``Hallmarks of Cancer'') that allow tumour cells to proliferate, survive and migrate \citep{Hanahan2011}.
There have been many reports on the association between obesity and worsened cancer prognosis, suggesting that there may be distinct changes that underlie tumorigenesis and/or cancer progression that are specific to obese patients.
The exact mechanism in which obesity accelerates cancer progression is unknown, but obesity is thought to disrupt the levels of various hormones (such as insulin, leptin, adeponectin, and other adipokines) that eventually activate tumorigenic pathways (for an overview, see the recent review by \citet{Renehan2015}).

\citet{Creighton2012} identified 662 genes that were able to distinguish breast tumour from obese patients from those of non-obese patients.
Furthermore, they showed that small molecules were able to induce or repress these genes, signifying the fact that these obesity specific genes could be used as an indicator for alternative or combinatorial chemotherapy and assist in better clinical decisions.

A more recent study by \citet{Fuentes-Mattei2014} showed that tumours in obese mice developed and progressed much faster than those in control mice.
Differential gene expression analysis showed that the genes overexpressed were the target of protein kinase B (AKT), and the activation of the AKT/mammalian target of rapamycin (mTOR) was observed.
Importantly, they were also able to control AKT/mTOR pathway activation by using inhibitors that targeted  downstream effectors.

Both studies showed that there were obesity specific genetic signatures in breast cancer, but can these genetic signatures be applied to other cancer types?
Other studies have shown that there are genetic signatures common in multiple cancer types \citep{Alexandrov2013,Lawrence2014}, however, these studies have not shown any evidence of obesity-specific genetic signatures that are common across multiple cancer types.

This research aims to determine whether gene expression signatures exist that are specific to obesity across multiple cancer types, and to investigate whether there are any common pathways being dysregulated in cancers based on these genetic signatures.
Better understanding of the pathways being dysregulated in cancer cells in obese patients may lead to improved clinical decisions, and could thus contribute towards personalised treatment in the future.

%\newpage

\vspace{-11pt}

\subsection*{Methods}

\begin{description}[leftmargin=0pt]

\item[Validation of the methodology used.]

The raw gene expression data files of all the samples used by \citet{Creighton2012}  were downloaded from the  Gene Expression Omnibus (GEO) database \citepalias{GEO}.
The microarray expression data for all 103 samples were loaded into R/RStudio and were RMA normalised using the  ``affy'' package.
The 799 obesity-related  gene probes were extracted from the data and Singular Value Decomposition was applied to this matrix to generate a metagene summary of these probes.
The metagene was compared with the BMI data of the samples which was plotted as a boxplot (BMI status) and a dotplot (BMI values).
The same method was applied to \citet{Fuentes-Mattei2014} data.

\item[Identification of samples with BMI data.]

The clinical data for all cancer types (33 in total) were downloaded from The Cancer Genome Atlas (TCGA) database and were checked to determine whether the samples had both height and weight data \citepalias{TCGA}.

\item[BMI calculation and categorisation.]

    BMI were calculated using the samples' height and weight data, where:
    \begin{equation*}
    BMI = \frac{Weight(kg)}{Height^2(m^2)}
\end{equation*}
Samples were then categorised into normal weight (BMI \textless{ }25), overweight (25 $\le$ BMI $<$ 30) and obese (BMI $\ge$ 30).

\item[Processing the cancer data.]

RNA-seq data for all of the cancer types with both height and weight data (8 in total) were downloaded from the International Cancer Genome Consortium (ICGC) database \citepalias{ICGC}.
All of these data were loaded into R/RStudio and were processed into a matrix form.
The data were then logged to the base 10, and standardised so that each gene had a mean of 0 and standard deviation of 1.

\item[Transforming the cancer data.]

A transformation matrix was used to obtain the metagene for the eight downloaded cancer datasets.
In order to transform the data, the transformation matrix from the original data must have the same number of genes as the new data, so the genes present in both the Creighton \textit{et al.} data and all of the other cancer types were identified.
The transformation matrix was made from the Creighton \textit{et al.} data using these common genes, and then this transformation matrix was used to transform all of the cancer types.

\item[Assessing whether the metagene is associated with BMI.]

Once the metagenes were obtained for each cancer type, the metagene was plotted against BMI status and raw BMI value.

\end{description}

%\newpage


\vspace{-30pt}

\subsection*{Results and Discussion}

Since the project was based on the results presented by \citet{Creighton2012}, the methods used in this project was applied to the raw data from their paper and the results were compared to validate whether the methodolgy to be used in this project is applicable to other data.

\begin{figure}[h!]
\centering
\includegraphics[width=0.6\textwidth]{creightonsvdheatmap}
\caption{\textbf{Heatmap of the gene expression of all the samples in Creighton \textit{et al.} data.}
Histogram on the top left hand corner shows the scale of the gene expression level, where red  and blue corresponds to high or low expression, respectively.
The single row on top of the heatmap represents the metagene produced from the singular value decomposition of the Creighton \textit{et al.} data.
The graph to the left of the heatmap is the correlation tree of the genes, and the labels to the right of the heatmap are the genes used.
The labels at the bottom of the heatmap are the sample ID, ordered by the metagene score (lowest score to the left, highest score to the right).}
\label{crheatmap}
\end{figure}

After the Creighton \textit{et al.} data were normalised using the RMA method, singular value decomposition was applied to the data to obtain the obesity metagene and plotted on a heatmap (Figure \ref{crheatmap}).
The resulting heatmap showed that the overall expression of the obesity-specific genes correlated with the metagene.
To assess whether the metagene was actually correlating with the sample BMI, the metagene was plotted against the sample BMI status and sample BMI value (Figure \ref{crmetavsbmi}).
These plots clearly showed that the metagene was able to identify and separate the obese samples from the normal and overweight samples.
Together, these plots confirmed that the methodology applied to the data set was able to recapitulate the results presented by \citet{Creighton2012}.

\begin{figure}[h!]
\centering
\includegraphics[width=0.4\textwidth]{crflipped}
\includegraphics[width=0.4\textwidth]{crflipped2}
\caption{\textbf{Box and dot plots of the Creighton \textit{et al.} metagene against BMI status and BMI value of the samples, respectively.}
The y-axis represents the metagene score and the x-axis represents the BMI status (for box plot) and BMI values (for dot plot).
The metagene was able to separate the obese samples from the normal/overweight samples, as shown in the box plot.
}
\label{crmetavsbmi}
\end{figure}

\begin{figure}[h!]
\centering
\includegraphics[width=0.47\textwidth]{crmetastd1}
\includegraphics[width=0.47\textwidth]{crmetastdclin1}
\caption{\textbf{Heatmap of the gene expression of all the sample with BMI data in bladder cancer (BLCA) data using the transformation matrix from Creighton \textit{et al.} data  (left) and a box plot of the metagene against BMI status of the samples (right).}
The parameters, scales and axes used in the heatmap are identical to that from Figure \ref{crheatmap}.
The overall gene expression of the samples were captured by the metagene, as shown by the heatmap, but there was no correlation with the BMI status of the samples.
Similar observations were made for other cancer types (data not shown).
}
\label{blcameta}
\end{figure}

The metagenes for other cancer types were generated by using the obesity-specific genes from breast cancer.
Transformation matrix was applied to each cancer type to generate the metagene and was plotted on the heatmap (Figure \ref{blcameta}).
All of the cancer types showed significant correlation with the metagene created from the transformation, where the high/low metagene values corresponded with high/low overall gene expression, respectively.

\begin{figure}[h!]
\centering
\includegraphics[width=0.47\textwidth]{fmmetaheat1}
\includegraphics[width=0.47\textwidth]{fmmetaclin1}
\caption{\textbf{Heatmap of the gene expression of all the sample with BMI data in bladder cancer (BLCA) data using the transformation matrix from Fuentes-Mattei \textit{et al.} data  (left) and a box plot of the metagene against BMI status of the samples (right).}
The parameters, scales and axes used in the heatmap are identical to that from Figure \ref{crheatmap}.
The overall gene expression of the samples were captured by the metagene, as shown by the heatmap, but there was no correlation with the BMI status of the samples.
Similar observations were made for other cancer types (data not shown).
}
\label{blcameta}
\end{figure}

To confirm whether the association was in fact due to BMI of the samples, plots of metagene against BMI status and BMI value were created.
In contrast to the  plots made from the Creighton \textit{et al.} data, no cancer types showed any significant separation between the obese samples and the normal/overweight samples (Figure \ref{blcameta}).
These results suggest that the obesity-specific genes from breast cancer were not specific enough to be used as a genetic marker for other cancer types.

The recent paper by \citet{Fuentes-Mattei2014} also found  obesity-specific genes (111 unique genes) in breast cancer, and these genes were also investigated.
Similar methods were applied to the Fuentes-Mattei \textit{et al.} data, and metagenes were obtained for each cancer type.
Again, the metagene seemed to correlate with the gene expression level of the cancer types, but there was no association with BMI value or BMI status.
However, it is interesting how the metagenes created from both Creighton \textit{et al.} and Fuentes-Mattei \textit{et al.} data were able to capture the overall gene expression profile, but not BMI.
It may be that the obesity-specific genes identified by both groups may have identified genes that were specific to a different clinical variable and/or molecular marker that was somehow related to the sample BMI, but not BMI itself.
Further analyses must be conducted to conclude these results with certainty.

Although current results do not show any significant correlation of the obesity-specific genes and the sample BMI in different cancers, there are many places for improvement.
%Firstly, the data processing of the raw RNA-seq data was carried out using a program written by myself which could have processed the data incorrectly.
Firstly, the data processing of the raw RNA-seq data was carried out using only one program, which may have processed the data incorrectly.
There are other programs to process these raw data, so the data processing procedure should be repeated and compared with the current results.

Secondly, the obesity-specific genes specific to each cancer type were not individually checked (i.e. per gene) with the BMI values of the samples.
The results presented here only showed the association of the metagene, created from a different cancer type and data (Creighton \textit{et al.} and Fuentes-Mattei \textit{et al.} data), with the BMI value and status, and it would be good to double-check whether the expression of the individual genes show any correlation with the BMI of the samples in other cancer types.

For future experiments, differential expression analysis for each cancer type should be carried out to identify the genes that are dysregulated in obese samples but not in the normal or overweight data.
These genes can then be compared with the genes from other cancer types to identify genes that are  commonly up or down regulated in different cancer types, which are specific to BMI status.


\vspace{-11pt}

%%Referencing:
\bibliographystyle{BiocRefStyle}
\bibliography{../../../References/BibTeX/MSc,../../../References/BibTeX/nonpapercitations}

\end{document}
