\documentclass[a4paper, 12pt]{article}

\usepackage[utf8]{inputenc}
\usepackage{geometry}

\geometry{
    a4paper,
    left=3cm,
    right=3cm,
    top=3cm,
    bottom=3cm
}

\linespread{1.3}

\begin{document}

\begin{center}
\large{Olfactory Receptors, Copy Number Variations and Evolution}
\end{center}

\section*{Abstract}

Human olfactory receptors (ORs) make up the largest family of genes in the genome, yet there has been little work done on characterising these receptors.
Studies have shown that OR genes are highly copy number variable within and between species, but no study has associated this with phenotypic characteristics.
Sensory properties have been linked to satiety and eating behaviours, but has not been described in terms of the individual genotype.
Next generation sequencing will be used to investigate the effect of OR copy number variation (CNV) on the individual phenotype, and to see whether this affects the sensory-specific satiety and/or food consumption.

\section*{Introduction}

\subsection*{Olfactory Receptors}

Olfaction, or the sense of smell, is mediated by a large group of sensory receptors known as the olfactory receptors (ORs).
ORs all belong to the G-protein coupled receptor (GPCR) superfamily like other sensory receptors, such as for vision and taste.
ORs are known to be the largest gene family in the human genome and form clusters on many chromosomes, and consists of about 900 genes of which at least half of these genes are pseudogenes and have no functional activity (ref Glusman et al 2001; Niimura and Nei 2005).

Due to the fact that there are so many OR genes in the human genome, and that these ORs work in a combinatorial manner, it has been difficult for the researchers to decipher the complete code of olfaction (ref Hasin-Brumshtein et al, 2009; Nei et al 2008).
In addition to this, ORs are highly copy number variable within and between species, adding greater complexity to the olfactory process and remains to be a challenge for the researchers to make a connection between OR CNV with the phenotype (ref Nei et al, 2007).

OR genes also have an important implications in the evolution of different species.
There are two major classes of OR genes: class I and class II genes, or ``fish-like'' and ``mammalian-like'' genes respectively, as the different classes of genes are found mainly in their respective species (ref original paper or Nei and Niimura 2005).
The study by Niimura and Nei (2005) showed that the diversity of the OR gene families in fishes was greater than that of the mammalian OR gene family, even though the number of OR genes in a single family were less in fishes.
Furthermore, they showed that amphibians have retained the diversity of the gene families as the fishes, while also having the large repertoire of OR genes in one family as in the mammals, signifying the fact that the CNV was likely due to the adaptation process from being aquatic to terrestial (Nei and niimura, 2005).

Another study by Liman (2006) showed that pseudogenization and CNVs of the ORs coincided with the trichromatic vision in primates.
Although this is an evidence of how trichromacy in primates became more important than the ORs for their survival, better understanding of the ligands of the ORs is crucial to support this theory.

%As both pseudogenization and CNV are evidence of genomic drift, it is of great interest for researchers to understand the evolutionary process of ORs by studying the OR CNV (ref Nei et al, 2007).

\subsection*{Next Generation Sequencing and Copy Number Variations}

Recent advances in next generation sequencing technology have allowed many researchers to investigate various aspects of the genome, ranging from something as small as single nucleotide polymorphisms (SNPs) to large chromosomal rearrangements (ref NGS papers).
Copy number variations (CNVs) is one such genomic rearrangement whereby some genes get duplicated, inserted, deleted and/or inverted.

studies have been relatively small -- possibly due to Sequencing limitations.
talk about 1000 genome project -- provides greater sample size to work with
limitations of sequencing CNVs -- highly variable, hard to sequence/annotate??

\subsection*{Intro to olfaction and sensory specific satiety}

relate this to obesity and eating disorder

\subsection*{Significance of the work}

evolution and healthier eating

\section*{Research Proposal}

gather sample to identify the OR genes that could be important -- get samples with known smelling disorder/condition
sequence the samples and identify the CNV difference and target those genes for further work
check 1000 genome database for any correlation and/or evidence of evolution

\section*{Conclusion}



\end{document}
