\documentclass[12pt, a4paper]{article}

\usepackage[utf8]{inputenc}
\usepackage{natbib}
\usepackage{geometry}
\usepackage{fixltx2e}
\usepackage{enumitem}
\usepackage{amsmath}

\geometry{
    a4paper,
    left=2.5cm,
    right=2.5cm,
    top=2.5cm,
    bottom=2.5cm
}

\linespread{1.3}

\begin{document}

\begin{center}
\Large{\textbf{Body Mass Index and Pathway Dysregulation in Cancer}}\\
\end{center}

\begin{center}
Riku Takei (MSc)\\
Supervisor: Assoc. Prof. Mik Black
\end{center}

\subsection*{Abstract (50-words)}

Recent studies in breast cancer have shown that there are genetic signatures that are specific to obesity, and these signatures were further investigated in other cancer types (Creighton and fuentes mattei paper).
The obesity genes identified by Creighton \textit{et al.} did not correlate with any of the cancer types investigated.

\subsection*{Introduction}



\newpage

\subsection*{Methods}

\begin{description}[leftmargin=0pt]

\item[Validation of the methodology used.]

The original CEL files of all the samples used by Creighton \textit{et al.} were downloaded from Gene Expression Omnibus (GEO) database (website ref).
The microarray expression data for all 103 samples were loaded into R/RStudio and were RMA normalised using the package '????'.
The 799 obesity gene probes were extracted from the data and singular value decomposition was applied to this matrix to generate a metagene of the obesity genes.
The metagene was compared with the BMI data of the samples which was plotted as a boxplot (BMI status) and a dotplot (BMI values).

\item[Identification of samples with BMI data.]

The clinical data for all cancer types (33 in total) were downloaded from The Cancer Genome Atlas (TCGA) database and were checked whether the samples had both height and weight data.

\item[BMI calculation and categorisation.]

    BMI were calculated using the samples' height and weight data, where:
    \begin{equation*}
    BMI = \frac{Weight(kg)}{Height^2(cm^2)}
\end{equation*}
Samples were then categorised into normal weight (BMI \textless{ }25), overweight (25 $\le$ BMI $<$ 30) and obese (BMI $\ge$ 30).

\item[Assessing whether the metagene is associated with BMI.]

Once the metagenes were obtained for each cancer type, the metagene was plotted against BMI status and raw BMI value.

\item[Processing the cancer data.]

RNA-seq data for all of the cancer types with both height and weight data (8 in total) were downloaded from the International Cancer Genome Consortium (ICGC).
All of these data were loaded into R/RStudio and were processed into a matrix form.
The data were then logged to the base 10, and standardised so that each gene had a mean of 0 and standard deviation of 1.

\item[Transforming the cancer data.]

Transformation matrix was used to obtain the metagene for the other cancer data.
In order to transform the data, the transformation matrix from the original data must have the same number of genes as the transforming data, so the genes present in both the Creighton \textit{et al.} data and all of the other cancer types were identified.
Transformation matrix was made from the Creighton \textit{et al.} data using these common genes, and then this transformation matrix was used to transform all of the cancer types.

\end{description}

\newpage

\subsection*{Results and Discussion}

Since the project was based on the results presented by Creighton \textit{et al.} (2012), that there were obesity-specific genetic signatures in breast cancer, the methods used in this project was applied to the raw data from their paper and the results were compared.


must not exceed 6 pages (with figures and references).

%\citep{Hanahan2011}

%%Referencing:
%\bibliographystyle{BiocRefStyle}
%\bibliography{../../../References/BibTeX/MSc}

\end{document}
