% \documentclass[notes]{beamer}
% \documentclass[notes=hide]{beamer}
% \documentclass[notes=only]{beamer}
\documentclass[handout]{beamer}

% \usepackage{pgfpages}
% \pgfpagesuselayout{2 on 1}[a4paper,border shrink=3mm]

\usepackage[utf8]{inputenc}
\usepackage[T1]{fontenc}
\usepackage{graphicx}
\usepackage{natbib}
\usepackage{xcolor,colortbl}

\definecolor{LightCyan}{rgb}{0.88,1,1}

\usetheme{riku}

\def\logo{
	{\includegraphics[width=20mm]{logo}}
}

\title[BMI and Cancer]{Body Mass Index and Pathway Dysregulation in Cancer}
\subtitle{Colloquia Talk}
\author[Riku Takei]{Riku Takei (MSc)\\Supervisor: Assoc. Prof. Mik Black}
\institute{Department of Biochemistry, University of Otago}

\begin{document}

{
	\setbeamertemplate{footline}{}
	\begin{frame}[noframenumbering]
		\titlepage
	\end{frame}
}

\begin{frame}{Overview}
	\begin{enumerate}
		\item Not so quick intro
		\item Part 1 - BMI genes and Cancer
		\item Part 2 - BMI genes and Pathways
	\end{enumerate}
\end{frame}

\begin{frame}
	\frametitle{Body Mass Index and Cancer}
	\begin{columns}
		\column{0.6\textwidth}
		{\footnotesize
			Cancers are caused by dysregulation in various pathways/mechanisms that allow them to survive and proliferate
			\begin{itemize}
				\item Commonly know as the Hallmarks of Cancer
			\end{itemize}
			Many reports on obesity being a major risk for cancer and poor prognosis
		}
		\column{0.5\textwidth}
		\includegraphics[width=0.45\paperwidth]{./images/cancerhallmark}
	\end{columns}
\end{frame}

\begin{frame}
	\frametitle{Obesity-associated Genetic Signatures 1}
	Creighton \textit{et al.} (2012)
	\begin{itemize}
		\item Identified obesity-associated genetic signatures from the tumours of breast cancer patients
		\item Showed that some compounds were able to specifically regulate the expression of these genetic signatures (potential for targeted therapy)
	\end{itemize}
\end{frame}

\begin{frame}{Validation of Methodology -- Creighton \textit{et al.} (2012)}
	103 samples used in the study, 799 obesity-associated gene probes (695 unique genes) identified\\
	{\footnotesize
		\begin{itemize}
			\item Majority of these genes were downregulated in obese patients
		\end{itemize}
	}
	\begin{center}
		\includegraphics[width=0.6\paperwidth]{./images/creighton}
	\end{center}
\end{frame}

\begin{frame}{Obesity-associated genes from Fuentes-Mattei \textit{et al.}}
	\begin{columns}
		\column{0.5\textwidth}
		Fuentes-Mattei \textit{et al.} (2014)
		\begin{itemize}
			\item 137 ER+ breast cancer patients in the study, 130 obesity-associated genes identified
			\item Direct evidence of how obesity/adiposity contributes to tumour growth (using mouse model)
		\end{itemize}
		\column{0.5\textwidth}
		\includegraphics[width=0.45\paperwidth]{./images/fmcircos}
	\end{columns}
\end{frame}

\begin{frame}
	\frametitle{Aims}
	\begin{enumerate}
		\item Investigate the association between patient BMI and tumour molecular characteristics
		\item Determine whether pathway commonalities exist across tumours that arise from other tissues
	\end{enumerate}
\end{frame}

\begin{frame}
	\frametitle{Aims}
	\begin{enumerate}
		\item Are there any genes that are specific to obese patients, and can we use it to identify those patients by using those gene?
		\item Are there any pathways common to multiple different cancer types?
	\end{enumerate}
\end{frame}

\begin{frame}
	\frametitle{Methods}
	Downloaded RNA-seq data from 8 different cancer types from ICGC
	\vskip2ex
	Microarray data from Creighton's paper
	\vskip2ex
	Microarray data from Fuentes-Mattei's paper
	\vskip2ex
	Microarray data from Cris Print (Mik's friend up in Auckland)
	\vskip2ex
	\includegraphics[width=0.4\textwidth]{./images/tcga}
	\hfill
	\includegraphics[width=0.4\textwidth]{./images/icgc}
\end{frame}

\begin{frame}{Clinical Data}
	You need clinical (BMI) data to check if the metagenes are actually working
	\vskip2ex
	ICGC, Creighton and Cris' data have BMI data, but FM data doesn't
	\vskip2ex
	\begin{equation*}
		BMI = \frac{Weight (kg)}{Height^2 (m^2)}
	\end{equation*}
\end{frame}

\begin{frame}{Clinical Data with Patient BMI}
	\begin{center}
		\begin{tabular}{lcccl}
			& Normal & Overweight & Obese & Total\\
			\rowcolor{LightCyan}
			BLCA & 103  & 97         & 61    & 261\\
			CESC & 85   & 62         & 77    & 224   \\
			\rowcolor{LightCyan}
			COAD & 75   & 80         & 71    & 226   \\
			KIRP & 26   & 53         & 45    & 124\\
			\rowcolor{LightCyan}
			LIHC & 140  & 75         & 49    & 264\\
			READ & 22   & 36         & 15    & 73\\
			\rowcolor{LightCyan}
			SKCM & 72   & 73         & 73    & 218\\
			UCEC & 90   & 106        & 286   & 482\\
			\hline
			\rule{0pt}{2.25ex}Total & 613 & 582 & 677 & 1872\\
		\end{tabular}
	\end{center}
\end{frame}

\begin{frame}{Validation of Methodology -- Creighton \textit{et al.} (2012)}
	103 samples used in the study, 799 obesity-associated gene probes (695 unique genes) identified\\
	\vskip2ex
	Metagene was obtained from SVD using the genes that were common in both Creighton \textit{et al.} and other cancer data sets\\
	\begin{center}
		\includegraphics[width=0.6\paperwidth]{./images/creighton}
	\end{center}
\end{frame}

\begin{frame}{Singular Value Decomposition (SVD) recap}
	For each sample in the data set, you need to assign a ``score'' that represents the overall gene expression of the gene set of interest
	\begin{itemize}
		\item i.e. a score that represent the 799 obesity-associated gene probes identified by Creighton \textit{et al.}
	\end{itemize}
	\vskip2ex
	You can do this by applying SVD to the data set of interest and get a metagene out from this
\end{frame}

\begin{frame}{\normalsize Creighton \textit{et al.} metagene correlates with Gene Expression}
	\begin{center}
		\includegraphics[width=0.45\paperwidth]{./images/crge1}
	\end{center}
\end{frame}

\begin{frame}{Creighton \textit{et al.} metagene correlates with BMI}
	\hspace{-4.0ex}
	\includegraphics[width=0.45\paperwidth]{./images/crmetaclin0}
	\includegraphics[width=0.45\paperwidth]{./images/crmetaclin1}
\end{frame}

\begin{frame}{Transformation Matrix}
	Singular Value Decomposition of a dataset X gives:
	\begin{equation*}
		X = UDV'
	\end{equation*}
	Rearranging this equation gives:
	\begin{equation*}
		V' = D^{-1}U'X
	\end{equation*}
	Where $D^{-1}U'$ is the transformation matrix and $V'$ is the metagene for the data set X
	{\footnotesize
		\begin{itemize}
			\item This means that the transformation matrix can be applied to other data sets to obtain the metagene for those data sets
		\end{itemize}
	}
\end{frame}

\begin{frame}{Metagene in ICGC data}
	\includegraphics[width=0.23\paperwidth]{./images/crtcga2}
	\includegraphics[width=0.23\paperwidth]{./images/crtcga7}
	\includegraphics[width=0.23\paperwidth]{./images/crtcga12}
	\includegraphics[width=0.23\paperwidth]{./images/crtcga17}\\
	\includegraphics[width=0.23\paperwidth]{./images/crtcga22}
	\includegraphics[width=0.23\paperwidth]{./images/crtcga27}
	\includegraphics[width=0.23\paperwidth]{./images/crtcga32}
	\includegraphics[width=0.23\paperwidth]{./images/crtcga37}
\end{frame}

\begin{frame}{Does the metagene correlate with BMI?}
	\includegraphics[width=0.23\paperwidth]{./images/crtcga3}
	\includegraphics[width=0.23\paperwidth]{./images/crtcga8}
	\includegraphics[width=0.23\paperwidth]{./images/crtcga13}
	\includegraphics[width=0.23\paperwidth]{./images/crtcga18}\\
	\includegraphics[width=0.23\paperwidth]{./images/crtcga23}
	\includegraphics[width=0.23\paperwidth]{./images/crtcga28}
	\includegraphics[width=0.23\paperwidth]{./images/crtcga33}
	\includegraphics[width=0.23\paperwidth]{./images/crtcga38}
\end{frame}

\begin{frame}{Obesity-associated genes from Fuentes-Mattei \textit{et al.}}
	\begin{columns}
		\column{0.5\textwidth}
		137 ER+ breast cancer samples in the study, 130 unique genes identified\\
		\vskip2ex
		Data were normalised  with RMA normalisation method\\
		\vskip2ex
		\column{0.5\textwidth}
		\includegraphics[width=0.45\paperwidth]{./images/fmcircos}
	\end{columns}
\end{frame}

\begin{frame}{FM metagene in Creighton \textit{et al.} data}
	\begin{center}
		\includegraphics[width=0.45\paperwidth]{./images/fmmetacr3}
	\end{center}
\end{frame}

\begin{frame}{FM metagene in Creighton \textit{et al.} data}
	\includegraphics[width=0.45\paperwidth]{./images/fmmetacr4}
	\includegraphics[width=0.45\paperwidth]{./images/fmmetacr5}
\end{frame}

\begin{frame}{FM metagene in other cancer types}
	\includegraphics[width=0.23\paperwidth]{./images/fmmetaheat2}
	\includegraphics[width=0.23\paperwidth]{./images/fmmetaheat7}
	\includegraphics[width=0.23\paperwidth]{./images/fmmetaheat12}
	\includegraphics[width=0.23\paperwidth]{./images/fmmetaheat17}\\
	\includegraphics[width=0.23\paperwidth]{./images/fmmetaheat22}
	\includegraphics[width=0.23\paperwidth]{./images/fmmetaheat27}
	\includegraphics[width=0.23\paperwidth]{./images/fmmetaheat32}
	\includegraphics[width=0.23\paperwidth]{./images/fmmetaheat37}
\end{frame}

\begin{frame}{FM metagene and BMI}
	\includegraphics[width=0.23\paperwidth]{./images/fmmetabmi3}
	\includegraphics[width=0.23\paperwidth]{./images/fmmetabmi8}
	\includegraphics[width=0.23\paperwidth]{./images/fmmetabmi13}
	\includegraphics[width=0.23\paperwidth]{./images/fmmetabmi18}\\
	\includegraphics[width=0.23\paperwidth]{./images/fmmetabmi23}
	\includegraphics[width=0.23\paperwidth]{./images/fmmetabmi28}
	\includegraphics[width=0.23\paperwidth]{./images/fmmetabmi33}
	\includegraphics[width=0.23\paperwidth]{./images/fmmetabmi38}
\end{frame}

\begin{frame}{Summary 1}
	Metagene from Creighton \textit{et al.} correlated with gene expression and patient BMI in Creighton data set
	\vskip2ex
	Metagene derived from Creighton's obesity-associated genes  correlated with the gene expression but not the patient BMI in ICGC data
	\vskip2ex
	Metagene derived from Fuentes-Mattei obesity-associated genes correlated with the gene expression but not the patient BMI in both Creighton \textit{et al.} and ICGC data
\end{frame}

\begin{frame}{Gene Expression Analysis}
	Maybe the obesity-associated genes identified by either group were not related to obesity, but related to a different clinical variable
	\vskip2ex
	Are the differentially expressed genes between the obese and non-obese groups (after correcting for other clinical variables) similar to the obesity-associated genes?
	{\footnotesize
		\begin{itemize}
			\item How do these DEGs compare with the Creighton's or Fuentes-Mattei's obesity-associated genes?
		\end{itemize}
	}
\end{frame}

\begin{frame}{Creighton \textit{et al.} data revisited}
	Gene expression analysis was carried out on the Creighton \textit{et al.} data
	{\footnotesize
		\begin{itemize}
			\item in their paper, they found 799 gene probes with p \textless{} 0.01 and fold change \textgreater{} 1.2
		\end{itemize}
	}
	\vskip2ex
	1781 differentially expressed genes (p \textless{} 0.01) were found between obese vs. non-obese
	{\footnotesize
		\begin{itemize}
			\item very few genes had fold change \textgreater{} 1.2 (132 genes)
			\item no genes were found when p-value was adjusted
			\item top 799 genes were chosen from this list
		\end{itemize}
	}
	\vskip2ex
	239 gene probes were present in the top 799 gene probes that I have found
\end{frame}

\begin{frame}{Residual data}
	All of the non-BMI clinical variables were controlled for in the linear model
	{\footnotesize
		\begin{itemize}
			\item Age, ethnicity, menopause status, grade, LN status, ER/PR/HER2 statuses
		\end{itemize}
	}
	\vskip2ex
	Residual data (or remaining data after controlling for the variables) was used to find the DEGs
	\vskip2ex
	Again, more than 799 gene probes showed up (1104 genes)
	{\footnotesize
		\begin{itemize}
			\item 168 genes from Creighton \textit{et al.} were found in the top 799 genes
		\end{itemize}
	}
\end{frame}

\begin{frame}{DEGs in residual data}
	\begin{center}
		\includegraphics[width=0.50\paperwidth]{./images/venn0}
	\end{center}
\end{frame}

\begin{frame}{Completely remove the effect of ethnicity}
	African American patients are more likely to have higher BMI than Cauacasian patients
	\vskip2ex
	Removed the effect of ethnicity by removing the African American patients' data (total of 77 patient data to work with)
	{\footnotesize
		\begin{itemize}
			\item All of the other clinical variables were adjusted as before
		\end{itemize}
	}
	\vskip2ex
	Top 799 genes were identified in this data set (both raw and residual)
	{\footnotesize
		\begin{itemize}
			\item  148 and 92 genes were in the top 799 genes in the raw and residual data, respectively
		\end{itemize}
	}
\end{frame}

\begin{frame}{DEGs in Cauacasian-only data}
	\begin{center}
		\includegraphics[width=0.50\paperwidth]{./images/venn1}
	\end{center}
\end{frame}

\begin{frame}{Metagene Analysis}
	All of the gene probe sets I have found (8 in total) were used to create metagenes and to transform other cancer types
	\begin{itemize}
		\item Checked in Creighton \textit{et al.} data
		\item Transferred into other cancer data sets
	\end{itemize}
\end{frame}

\begin{frame}{Initial results -- Checking in Creighton \textit{et al.} data}
	\includegraphics[width=0.23\paperwidth]{./images/degmetacr2}
	\includegraphics[width=0.23\paperwidth]{./images/degmetacr7}
	\includegraphics[width=0.23\paperwidth]{./images/degmetacr12}
	\includegraphics[width=0.23\paperwidth]{./images/degmetacr17}\\
	\includegraphics[width=0.23\paperwidth]{./images/degmetacr22}
	\includegraphics[width=0.23\paperwidth]{./images/degmetacr27}
	\includegraphics[width=0.23\paperwidth]{./images/degmetacr32}
	\includegraphics[width=0.23\paperwidth]{./images/degmetacr37}
\end{frame}

\begin{frame}{Initial results -- Checking in Creighton \textit{et al.} data}
	\includegraphics[width=0.23\paperwidth]{./images/degmetacr3}
	\includegraphics[width=0.23\paperwidth]{./images/degmetacr8}
	\includegraphics[width=0.23\paperwidth]{./images/degmetacr13}
	\includegraphics[width=0.23\paperwidth]{./images/degmetacr18}\\
	\includegraphics[width=0.23\paperwidth]{./images/degmetacr23}
	\includegraphics[width=0.23\paperwidth]{./images/degmetacr28}
	\includegraphics[width=0.23\paperwidth]{./images/degmetacr33}
	\includegraphics[width=0.23\paperwidth]{./images/degmetacr38}
\end{frame}

\begin{frame}{Do the metagenes correlate with GE in ICGC data?}
	\includegraphics[width=0.23\paperwidth]{./images/rawobsmeta2}
	\includegraphics[width=0.23\paperwidth]{./images/resobsmeta2}
	\includegraphics[width=0.23\paperwidth]{./images/caobsmeta2}
	\includegraphics[width=0.23\paperwidth]{./images/caresobsmeta2}\\
	\includegraphics[width=0.23\paperwidth]{./images/crolmeta2}
	\includegraphics[width=0.23\paperwidth]{./images/rescrolmeta2}
	\includegraphics[width=0.23\paperwidth]{./images/cacrolmeta2}
	\includegraphics[width=0.23\paperwidth]{./images/carescrolmeta2}
\end{frame}

\begin{frame}{Do the metagenes correlate with BMI?}
	\includegraphics[width=0.23\paperwidth]{./images/rawobsclin3}
	\includegraphics[width=0.23\paperwidth]{./images/resobsclin3}
	\includegraphics[width=0.23\paperwidth]{./images/caobsclin3}
	\includegraphics[width=0.23\paperwidth]{./images/caresobsclin3}\\
	\includegraphics[width=0.23\paperwidth]{./images/crolclin3}
	\includegraphics[width=0.23\paperwidth]{./images/rescrolclin3}
	\includegraphics[width=0.23\paperwidth]{./images/cacrolclin3}
	\includegraphics[width=0.23\paperwidth]{./images/carescrolclin3}
\end{frame}

\begin{frame}
	Repeated this with Cris' data, but same results as in Creighton and ICGC data
\end{frame}

\begin{frame}{Do the metagenes correlate with each other?}
	\begin{center}
		\includegraphics[width=0.45\paperwidth]{./images/cor1}
	\end{center}
\end{frame}

\begin{frame}{Metagenes are correlating with GE but not BMI}
	All of the metagenes found so far have correlated with the overall gene expression of the samples, but not with the sample BMI
	\vskip2ex
	The metagenes are picking up gene signatures  that are common across all cancer types, but not related to BMI
	\vskip2ex
	\begin{center}
		\textbf{So what signature/mechanism is the metagene picking up?}
	\end{center}
\end{frame}

\begin{frame}{Common genes across multiple cancer types}
	DEGs between the obese and non-obese groups were identified for each cancer types
	\vskip2ex
	For each cancer types, samples were randomly split into two groups (same proportion as obese vs. non-obese samples)
	\vskip2ex
	{\footnotesize
		\begin{tabular}{ccccccccc}
			& \multicolumn{8}{c}{No. of cancer types the gene overlapped}\\
			\rule{0pt}{2.25ex}                    & 1    & 2    & 3   & 4  & 5 & 6 & 7 & 8 \\
			\hline
			\rule{0pt}{2.25ex}Obese vs. Non-obese & 6865 & 1907 & 289  & 26   & 1 & 0 & 0 & 0 \\
			Randomly split groups                 & 6202 & 1488 & 198  & 18   & 1 & 0 & 0 & 0 \\
			No. of genes found                    & 5012 & 870  & 89   & 6    & 0 & 0 & 0 & 0 \\
			on average                            &      &      &      &      &   &   &   & \\
			Empirical FDR                         & 0.73 & 0.46 & 0.31 & 0.23 & 0 & 0 & 0 & 0 \\
		\end{tabular}
	}
\end{frame}

\begin{frame}{Pathway Enrichment Analysis}
	Maybe each cancer type have a different set of genes activated
	\vskip2ex
	Are there any specific pathways enriched in any specific cancer type?
	\vskip2ex
	If so, is it common across other cancer types?
	\vskip2ex
	How relevant is the pathway to obesity and/or cancer?
	\vskip2ex
	Can we use it to narrow down our search for obesity-associated signature?
\end{frame}

\begin{frame}{Pathway Enrichment Analysis}
	Gene Ontology (GO) database was used to check for the function(s) being enriched by the DEGs in each cancer type (between obese and non-obese groups)
	\vskip2ex
	Used a rank-based approach and controlled the error rate using FDR
	\vskip2ex
	\begin{center}
		\textbf {No pathways showed up as significantly enriched}
	\end{center}
\end{frame}

\begin{frame}{Gatza \textit{et al.} (2010)}
	Used 18 pathways to define sub-class of breast tumours, based on the expression of the pathway signatures
	\begin{itemize}
		\item AKT, ER, PR, HER2, PI3K, p53, MYC, RAS, etc...
	\end{itemize}
	\vskip2ex
	Each pathway signature was obtained by overexpressing a single gene and looking for differentially expressed genes
	\vskip2ex
	\begin{center}
		\textbf{I can use these pathways to see what the obesity-associated genes are most similar to}
	\end{center}
\end{frame}

\begin{frame}{Tricky thing about metagenes}
	When you do SVD, it gives each sample a summary score of the group of genes
	\vskip2ex
	Yes, the values are related to the group of genes, but not necessarily with the \textbf{direction} of the gene expression of interest
\end{frame}

\begin{frame}{What I mean about directions...}
	% TODO insert ER/PR or whatever gene with high correlation with metagene
	\includegraphics[width=0.45\paperwidth]{./images/output-0}
	\includegraphics[width=0.45\paperwidth]{./images/output-1}
\end{frame}

\begin{frame}{Some retarded genes}
	% TODO add some genes that are not correlated with its metagene
	\includegraphics[width=0.45\paperwidth]{./images/output-2}
	\includegraphics[width=0.45\paperwidth]{./images/output-3}
\end{frame}

\begin{frame}{Re-making results from Gatza \textit{et al.}}
	\includegraphics[width=0.9\paperwidth]{./images/gatzares2}
\end{frame}

\begin{frame}{Re-making results from Gatza \textit{et al.}}
	% TODO add  my result, beside Gatza's
	\includegraphics[width=0.45\paperwidth]{./images/out}
	\includegraphics[width=0.45\paperwidth]{./images/gatzares1}
\end{frame}

\begin{frame}{Re-making results from Gatza \textit{et al.}}
	% TODO add  my result, beside Gatza's
	\includegraphics[width=0.45\paperwidth]{./images/out2}
	\includegraphics[width=0.45\paperwidth]{./images/gatzares1}
\end{frame}

\begin{frame}{Problems associated with re-making the results}
	\begin{enumerate}
		\item Normalisation methods (RMA vs MAS5)
		\item Data scaling
		\item Metagene scaling (probit vs rank-based)
	\end{enumerate}
	\vskip2ex
	So I had to consider 8 different ways to figure out the direction of the metagenes
\end{frame}

\begin{frame}{Summary 2}
	Metagenes found in Creighton \textit{et al.} data correlates with GE and BMI status in Creighton data, but not in other data
	\vskip2ex
	There were no commonly expressed genes or pathways across different cancer types
	\vskip2ex
	Currently using Gatza's pathway metagenes to see which pathway(s) the obesity-associated genes cluster/group together with
	\vskip2ex
\end{frame}

\begin{frame}{Overall Summary}
	\textbf{Creighton/FM metagene analysis}
	\begin{itemize}
		\item Both Creighton and FM metagenes correlated with the overall gene expression in any cancer data set
		\item The metagene score does not correlate with the BMI status of the samples
	\end{itemize}
	\textbf{Gene expression and metagene analysis}
	\begin{itemize}
		\item Again, all of the gene sets found correlated with the overall gene expression in any data set, but none of them correlated with the BMI status
	\end{itemize}
	\textbf{Pathway enrichment analysis}
	\begin{itemize}
		\item There were no pathway significantly enriched in any cancer type
	\end{itemize}
\end{frame}

% \begin{frame}{Discussion}
% 	Tumour heterogeneity within and between samples
% 	{\footnotesize
% 		\begin{itemize}
% 				% TODO rewrite the following sentence
% 			\item Not finding any significant genes or pathways because the samples are so different from one another
% 		\end{itemize}
% 	}
% 	\vskip2ex
% 	Tumour sample quality and size
% 	{\footnotesize
% 		\begin{itemize}
% 			\item Picking up too much noise and/or don't have enough samples to find anything significant (i.e. shit data)
% 		\end{itemize}
% 	}
% \end{frame}

% \begin{frame}{Discussion -- Inter-tumour heterogeneity 1}
% 	Each patient has an obesity driven pathway dysregulated, as well as different ``core'' pathways that lead to cancer progression
% 	\vskip2ex
% 	Each data set will have a different collection of patients, which in turn will have a different collection of dysregulated pathways
% 	{\footnotesize
% 		\begin{itemize}
% 			\item Hence DEGs show up, but not the pathways
% 		\end{itemize}
% 	}
% \end{frame}

% \begin{frame}{Discussion -- Inter-tumour heterogeneity 2}
% 	Some of the high BMI patients in the data set will have tumours that are BMI driven, but BMI may not be the driver for some of these patients
% 	\begin{itemize}
% 		\item Only subset of the patients with high BMI are actually driven by BMI
% 	\end{itemize}
% 	\vskip2ex
% 	Other data sets may have less of these truly BMI driven tumours
% 	\begin{itemize}
% 		\item Therefore no association with the signatures
% 	\end{itemize}
% \end{frame}

\begin{frame}{Conclusion}
	Obesity-associated genetic signatures cannot be used in other cancer types, nor in the same cancer type with different sample population
	\vskip2ex
	Gene expression and pathway enrichment analyses showed that there may not be any obesity-associated signature across multiple cancer types
	\vskip2ex
	{\tiny Results from bioinformatics papers are unreliable, especially if the methodology heavily relies on third-party software and/or Excel}
\end{frame}

\begin{frame}{Acknowledgement}
	\begin{columns}
		\column{0.5\textwidth}
		My supervisor: Mik\\
		Members of the Black lab\\
		Cancer Genetics Lab\\
		SYSKA/Mozilla Study Group\\
		ICGC and TCGA\\
		\column{0.5\textwidth}
		\includegraphics[width=0.3\paperwidth]{./images/tcga}\\
		\includegraphics[width=0.3\paperwidth]{./images/icgc}
	\end{columns}
\end{frame}

\begin{frame}
	\frametitle{References}
	\bibliographystyle{BiocRefStyle}
	\bibliography{/home/riku/Documents/References/BibTeX/MSc}
\end{frame}

\begin{frame}{test}
	\citet{Fuentes-Mattei2014}\\
	\citet{Creighton2012}\\
	\citet{Alexandrov2013}
	\cite{Gatza2010a}
\end{frame}

\end{document}
